client/app/scripts/charts/nodes-chart.js

```diff
@@ -356,7 +353,7 @@ export default class NodesChart extends React.Component {
 
     // inject metrics and save coordinates for restore
```

Comment:
Needs comment on why we're re-applying the state nodes, a la "reapply in case layout runner's node cache applied stale node metadata".